### ISSUE TITLE
Longclick on marker without OnclickListener set

### ISSUE THREAD

**Issue by nilson-aguiar** [sababado/CircularView#7]
If you try to longclick on a marker without setting the click listener the app crashes

The ** **  part it's the suggestion to fix (CircularView.java)

```
private OnLongClickListener mOnLongClickListener = new OnLongClickListener() {
        @Override
        public boolean onLongClick(View v) {
            if ( **mOnCircularViewObjectClickListener != null** && isLongClickable()) {
                mLongClickRegistered = true;
                if (mTouchEventMarkerPos == -1) {
                    playSoundEffect(SoundEffectConstants.CLICK);
                    mOnCircularViewObjectClickListener.onClick(CircularView.this, true);
                    mTouchEventMarkerPos = null;
                } else if (mTouchEventMarker != null) {
                    playSoundEffect(SoundEffectConstants.CLICK);
                    mOnCircularViewObjectClickListener.onMarkerClick(CircularView.this, mTouchEventMarker, mTouchEventMarkerPos, true);
                    mTouchEventMarker = null;
                    mTouchEventMarkerPos = null;
                }
                return mLongClickRegistered;
            }
            return false;
        }
    };
```

**Comment by sababado** [comment]
Hey good catch. If you'd like to do a pull request feel free. Otherwise in a few hours I can make the change and push a 1.1.1

**Comment by sababado** [comment]
Issue fixed in commit: d6f28b5e010fc2b3fcb673e7f4309f2cff7b86be
v1.1.1 is being published now and should be available in a few hours (processing time)
